generator/recipe.go

```diff
@@ -93,6 +94,14 @@ func indent(spaces int, v string) string {
 	return pad + strings.Replace(v, "\n", "\n"+pad, -1)
 }
 
+func fmtSampleConfig(s string) string {
```

Comment:
@sudo-suhas lets rename this to something short? maybe `cfmt` ?

Reply:
Does rawfmt work?

Reply:
addressed in https://github.com/odpf/meteor/pull/412, please check